lib/opencensus/trace/span_context.rb

```diff
@@ -190,6 +228,22 @@ def this_span
         get_span span_id
       end
 
+      ##
+      # Builds all finished spans under this context, and returns an array of
+      # built `Span` objects. Ignores any unfinished spans. The order of the
+      # generated spans is undefined.
+      #
+      # Does not build any ancestor spans. If you want the entire span tree
+      # built, call this method on the `#root` context.
+      #
+      # @return [Array<Span>] Built Span objects
+      #
+      def build_contained_spans
+        contained_span_builders
+          .map { |sb| sb.finished? ? sb.to_span : nil }
```

Comment:
Map with a ternary and compact seems a bit odd. Maybe `filter` + `map` or `inject`?

Reply:
Changed to find_all + map